docs/api/instruments/keithley/index.rst

```diff
@@ -18,6 +18,22 @@ This section contains specific documentation on the Keithley instruments that ar
    keithley2450
    keithley2510
    keithley2600
+   keithley2601
+   keithley2602
+   keithley2611
+   keithley2612
+   keithley2600A
+   keithley2601A
+   keithley2602A
+   keithley2611A
+   keithley2612A
+   keithley2600B
+   keithley2601B
+   keithley2602B
+   keithley2604B
+   keithley2611B
+   keithley2612B
+   keithley2634B
```

Comment:
These names refer to files (with the suffix .rst) containing documentation.
As you do not add these files, these will lead to void.
Actually, as you do keep the keithley2600.py file, you should not add here any additional files, but add the new classes to the keithley2600.rst file.